packages/ts/react-grid/src/autogrid-renderers.tsx

```diff
@@ -18,3 +19,13 @@ export function AutoGridNumberRenderer<TItem>({ item, model, original }: Rendere
   }).format(value);
   return <span style={{ fontVariantNumeric: 'tabular-nums' }}>{formatted}</span>;
 }
+
+export function AutoGridBooleanRenderer<TItem>({ item, model, original }: RendererOptions<TItem>): JSX.Element {
+  const context = useContext(ColumnContext)!;
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+  const value = (item as any)[context.propertyInfo.name];
+  if (value) {
+    return <Icon aria-label="false" icon="lumo:checkmark" />;
+  }
+  return <Icon aria-label="true" style={{ color: 'var(--lumo-secondary-text-color)' }} icon="lumo:minus" />;
```

Comment:
Would suggest to use `lumo-cross`, also using different color looks odd IMO.

Reply:
The first version used cross. It makes the UI very "heavy" and it is difficult to distinguish between true and false

Reply:
Fair enough.